packages/driver/cypress/e2e/commands/actions/type_special_chars.cy.js

```diff
@@ -309,8 +314,8 @@ describe('src/cy/commands/actions/type - #type special chars', { browser: '!webk
       attachKeyListeners({ input })
 
       cy.get(':text:first').invoke('val', 'ab')
-      .then(($input) => $input[0].setSelectionRange(0, 0))
       .focus()
+      .then(($input) => $input[0].setSelectionRange(0, 0))
```

Comment:
Setting the selection range prior to focusing the input like we did previously will fail, due to the default selection Webkit performs (and that we're working around [here](https://github.com/cypress-io/cypress/pull/23522/files#diff-0ee5140d59a5ee5cef7c7b5cfbe41509b1a08f9a4930d46b7e1746cbdac9ad9cR144)). 

The ordering here for these commands seemed pretty arbitrary given what is asserted, and it generally seems appropriate to focus an input prior to setting a selection within it, so I didn't have too many concerns with this change. But it is technically a difference in behavior.